index.html

```diff
@@ -2587,26 +2587,29 @@ <h3>
           <a>purpose</a>.
         </p>
         <p class="note">
-          For example, an icon with purpose "<a>badge</a>" could be used as a
-          badge or pinned icon that is visually distinct, in color or form,
-          from an application's launch icon. The user agent uses the value of
-          the <a>purpose</a> member as a hint to determine where and how an
-          <a>ImageResource</a> is displayed. Unless declared otherwise by the
-          developer, a user agent can use an icon for <a>any purpose</a>.
+          For example, an icon with purpose "<a>single-color</a>" could be used
+          as a badge or pinned icon with a single solid color, visually
+          distinct from an application's full color launch icon. The user
+          agent uses the value of the <a>purpose</a> member as a hint to
+          determine where and how an <a>ImageResource</a> is displayed. Unless
+          declared otherwise by the developer, a user agent can use an icon for
+          <a>any purpose</a>.
         </p>
         <p>
           The <dfn>icon purposes</dfn> are as follows:
         </p>
         <dl>
           <dt>
-            <dfn data-lt="badge purpose">badge</dfn>:
+            <dfn data-lt="single-color purpose">single-color</dfn>:
           </dt>
           <dd>
-            A user agent can present this icon where space constraints and/or
-            color requirements differ from those of the application icon.
+            A user agent can present this icon where an icon with a
+            <a>single color</a> is needed. The
+            color information in the icon is discarded and only the alpha data
+            is used.
```

Comment:
So it is basically being used as a mask that can be used with any color by the user agent/platform. I think it might make sense to spell that out